marklogic-data-hub-central/ui/src/components/confirmation-modal/confirmation-modal.test.tsx

```diff
@@ -44,5 +45,124 @@ describe('Confirmation Modal Component', () => {
     userEvent.click(getByText('No'));
     expect(toggleModal).toBeCalledTimes(1);
   });
+
+  test('can render delete type confirmation', () => {
+    let entityName = 'Product';
+    let toggleModal = jest.fn();
+    let confirmAction = jest.fn();
```

Comment:
A trivial refactoring suggestion. You can define a noOpFunction with a global scope for tests to use as props.
let noOpFunction = jest.fn();//global context
toggleModal={noOpFunction}//props in test